ChangeLog

```diff
@@ -2,6 +2,7 @@ Changes in squid-6.7 (4 Feb 2024)
 
 	- Bug 5274: Successful tunnels logged as TCP_TUNNEL/500
 	- Fix crash when NTLM and Negotiate helpers are queried with no HTTP request
+	- Fix memory leak on SslBump certificates with Authority Key Identifier extension
```

Comment:
IIRC, we do not normally use an "SslBump certificate" term. I do not insist on any changes or any specific fix here, but can suggest this variation:

```suggestion
	- Fix SslBump memory leak when mimicking certificates with Authority Key Identifier
```

or, slightly more detailed/longer:

```suggestion
	- Fix SslBump memory leak when mimicking certificates with Authority Key Identifier extension
```